components/fxa-client/src/internal/state_manager.rs

```diff
@@ -220,6 +220,19 @@ impl StateManager {
         self.flow_store.clear();
     }
 
+    /// Called when we begin an OAuth flow.
+    ///
+    /// This clears out tokens/keys set from the previous time we completed an oauth flow.  In
+    /// particular, it clears the session token to workaround
```

Comment:
"workaround" seems the wrong phrase to use here - isn't it more like "to fix" or "to avoid" etc?